.rubocop_todo.yml

```diff
@@ -1,526 +1,505 @@
+    - 'lib/vmpooler/pool_manager.rb'
+
+# Offense count: 7
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles.
+# SupportedStyles: space, no_space
+Style/SpaceAroundEqualsInParameterDefault:
+  Exclude:
+    - 'lib/vmpooler/api/helpers.rb'
+    - 'lib/vmpooler/pool_manager.rb'
+    - 'lib/vmpooler/vsphere_helper.rb'
+
+# Offense count: 1
+# Cop supports --auto-correct.
+Style/SpaceAroundKeyword:
+  Exclude:
+    - 'lib/vmpooler/api/v1.rb'
+
+# Offense count: 4
+# Cop supports --auto-correct.
+# Configuration parameters: AllowForAlignment.
+Style/SpaceAroundOperators:
+  Exclude:
+    - 'lib/vmpooler/api/v1.rb'
+    - 'lib/vmpooler/statsd.rb'
+
+# Offense count: 4
+# Cop supports --auto-correct.
+Style/SpaceInsideBrackets:
+  Exclude:
+    - 'lib/vmpooler/api/v1.rb'
+    - 'lib/vmpooler/vsphere_helper.rb'
+
+# Offense count: 8
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles, EnforcedStyleForEmptyBraces, SupportedStylesForEmptyBraces.
+# SupportedStyles: space, no_space, compact
+# SupportedStylesForEmptyBraces: space, no_space
+Style/SpaceInsideHashLiteralBraces:
+  Exclude:
+    - 'lib/vmpooler/api/helpers.rb'
+
+# Offense count: 1
+# Cop supports --auto-correct.
+Style/SpaceInsideParens:
+  Exclude:
+    - 'lib/vmpooler/api/v1.rb'
+
+# Offense count: 2
+# Cop supports --auto-correct.
+Style/SpaceInsidePercentLiteralDelimiters:
+  Exclude:
+    - 'lib/vmpooler/api.rb'
+
+# Offense count: 47
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles, ConsistentQuotesInMultiline.
+# SupportedStyles: single_quotes, double_quotes
+Style/StringLiterals:
+  Exclude:
+    - 'lib/vmpooler/api.rb'
+    - 'lib/vmpooler/api/dashboard.rb'
+    - 'lib/vmpooler/api/reroute.rb'
+    - 'lib/vmpooler/api/v1.rb'
+    - 'lib/vmpooler/graphite.rb'
+    - 'lib/vmpooler/pool_manager.rb'
+    - 'lib/vmpooler/statsd.rb'
+    - 'lib/vmpooler/vsphere_helper.rb'
+    - 'vmpooler'
+
+# Offense count: 2
+# Cop supports --auto-correct.
+# Configuration parameters: IgnoredMethods.
+# IgnoredMethods: respond_to, define_method
+Style/SymbolProc:
+  Exclude:
+    - 'vmpooler'
+
+# Offense count: 1
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles, AllowSafeAssignment.
+# SupportedStyles: require_parentheses, require_no_parentheses, require_parentheses_when_complex
+Style/TernaryParentheses:
+  Exclude:
+    - 'lib/vmpooler/api/helpers.rb'
+
+# Offense count: 10
+# Configuration parameters: EnforcedStyle, SupportedStyles.
+# SupportedStyles: snake_case, camelCase
+Style/VariableName:
+  Exclude:
+    - 'lib/vmpooler/api/v1.rb'
+    - 'lib/vmpooler/pool_manager.rb'
+    - 'lib/vmpooler/vsphere_helper.rb'
+
+# Offense count: 2
+# Cop supports --auto-correct.
+# Configuration parameters: SupportedStyles, WordRegex.
+# SupportedStyles: percent, brackets
+Style/WordArray:
+  EnforcedStyle: percent
+  MinSize: -Infinity

```

Comment:
Well...looks like the autogeneration on Windows creates a CRLF file which git _mostly_ converts to LF for me when on Windows.  WIll have to keep an eye on that.